lua/lspconfig/fsautocomplete.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local bin_path = 'C:/tools/fsac/fsautocomplete.dll'
+if vim.fn.has('win32') == 0 then
+  bin_path = os.getenv('HOME')..'/lib/fsac/fsautocomplete.dll'
+end
+
+configs.fsautocomplete = {
+  default_config = {
+    cmd = {'dotnet', bin_path, '--background-service-enabled'};
+    root_dir = util.root_pattern('*.sln', '*.fsproj', '.git');
+    filetypes = {'fsharp'};
+    init_options = {
+      AutomaticWorkspaceInit = true;
+    };
+    settings = {};
+  };
+  docs = {
+    description = [[
+https://github.com/fsharp/FsAutoComplete
+
+Language Server for F# provded by FsAutoComplete (FSAC).
+
+Download a release of FsAutoComplete from [here](https://github.com/fsharp/FsAutoComplete/releases).
+It can also be built from sources. [Instructions](https://github.com/fsharp/FsAutoComplete) are on the main page.
+
+FsAutoComplete requires the [dotnet-sdk](https://dotnet.microsoft.com/download) to be installed.
+
+You may also need to configure the filetype as Vim defaults to Forth for `*.fs` files
+(unless you use something like vim-polyglot):
+
+`autocmd BufNewFile,BufRead *.fs,*.fsx,*.fsi set filetype=fsharp`
+
+This is automatically done by plugins such https://github.com/PhilT/vim-fsharp or
+https://github.com/fsharp/vim-fsharp.
+
+This config will look in the following directories for the FsAutoComplete.dll:
+
+* Default path on Windows: `C:\tools\fsac\fsautocomplete.dll`
+* Default path on Linux: `$HOME/lib/fsac/fsautocomplete.dll`
+
+If you wish to install to a different location set `cmd` as follows:
+
+```lua
+require'lspconfig'.fsautocomplete.setup{
+  cmd = {'dotnet', 'path/to/fsautocomplete.dll', '--background-service-enabled'}
+}
+```
+    ]];
+    default_config = {
+      cmd = [[{'dotnet', '[C:\tools|/$HOME/lib]/fsac/fsautocomplete.dll', '--background-service-enabled'}]];
```

Comment:
Default config is prepopulated by docgen, so you don't need to duplicate the above settings here

Reply:
sweet!